tests/kafkatest/services/templates/verifiable_producer.properties

```diff
@@ -0,0 +1,16 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+request.timeout.ms={{ request_timeout_ms }}
```

Comment:
Would it be possible to use `services/templates/producer.properties` instead of creating a new template? We could provide a default like the following:
```
request.timeout.ms={{ request_timeout_ms | default(30000) }}
```

Reply:
Great catch. Seems like `producer.properties` is not used at all so we can edit it as we wish